src/monitor_endpoint.py

```diff
@@ -82,7 +99,21 @@ def monitor_info():
     for balance in exchange_balances or []:
         info[f'exchange_{balance.symbol}_available'] = float(format(balance.available, f'.{balance.decimals}f'))
         info[f'exchange_{balance.symbol}_total'] = float(format(balance.total, f'.{balance.decimals}f'))
+        if balance.symbol == 'NZD':
+            if balance.available < EXCHANGE_MIN['NZD']:
+                info['exchange_NZD_available_threshold_count'] = 1
+            else:
+                info['exchange_NZD_available_threshold_count'] = 0
+        if balance.symbol == 'BTC':
+            if balance.available < EXCHANGE_MIN['BTC']:
+                info['exchange_BTC_available_threshold_count'] = 1
+            else:
+                info['exchange_BTC_available_threshold_count'] = 0
     info[f'bank_{crown_currency}_available'] = crown_balance
+    if crown_balance < BANK_MIN['NZD']:
+        info['bank_NZD_available_threshold_count'] = 1
+    else:
+        info['bank_NZD_available_threshold_count'] = 0
```

Comment:
```suggestion
    if crown_currency in BANK_MIN and crown_balance < BANK_MIN[crown_currency]:
        info[f'bank_{crown_currency}_below_min'] = 1
    else:
        info[f'bank_{crown_currency}_below_min'] = 0
```

dont hardcode accesses to `BANK_MIN` that arent guaranteed to be there

Reply:
made the changes above